examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy

```diff
@@ -44,6 +43,21 @@ class BulkRestApiSpec extends Specification implements OkHttpRestTrait, JsonPars
         body.data[0].data.num == "foox1"
         body.data[0].data.name == "Foox1"
 
+        when: "Verify job.data"
+        Job job = Job.get(body.id as Long)
+
+        then:
+        job != null
+        job.data != null
+        job.state == JobState.Finished
+
+        when: "Verify job.data json, this is what come in from the request"
+        List dataList = parseJson(job.requestData)
+
+        then:
+        dataList.size() == 3
+        dataList[0].num == "foox1"
+
```

Comment:
@snimavat can we test source and sourceid here ?

Reply:
Done